community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/suspend.py

```diff
@@ -87,16 +87,9 @@ def delete_tpu_instances(instances):
 
 def delete_instances(instances):
     """delete instances individually"""
-    invalid, valid = separate(lambda inst: bool(lkp.instance(inst)), instances)
```

Comment:
why this is removed ?

Reply:
This is one is a bit of stretch. Reverted to reduce the scope.